main.py

```diff
@@ -313,11 +313,12 @@ def main(cl_arguments):
                            "they should not be updated! Check sep_embs_for_skip flag or make an issue.")
 
         for task in target_tasks:
-            # Skip mnli-diagnostic
+            # Skip tuning on a target task, if it does not really have a training set (like mnli-diagnostic)
             # This has to be handled differently than probing tasks because probing tasks require the "is_probing_task"
             # to be set to True. For mnli-diagnostic this flag will be False because it is part of GLUE and
             # "is_probing_task is global flag specific to a run, not to a task.
-            if task.name == 'mnli-diagnostic':
+            if task.get_split_text('train') == task.get_split_text('val') or \
+                task.get_split_text('train') == task.get_split_text('test'):
```

Comment:
If this is at all slow, we should just set an attribute of tasks that we can check for.

Reply:
done